server/cmd/mmctl/commands/import.go

```diff
@@ -271,7 +271,7 @@ func importProcessCmdF(c client.Client, command *cobra.Command, args []string) e
 		Type: model.JobTypeImportProcess,
 		Data: map[string]string{
 			"import_file": importFile,
-			"local_mode":  strconv.FormatBool(isLocal && !isLegacy),
+			"local_mode":  strconv.FormatBool(isLocal && bypassUpload),
```

Comment:
I wonder if we should instead have this be `"bypass_upload"` instead of `"local_mode"`,  since that is what this actually signals

Reply:
My reasoning was that if were to do this from scratch, we would have implemented this behavior by default in local mode. So that's why I used `local_mode` on the server side. `bypass_upload` makes a bit more sense from the client perspective, as you are not uploading anything. But on the server side, `local_mode` would mean to just read the file locally which is what it's doing. Does that make sense?

Happy to change it as well if you feel strongly. LMK.

Reply:
> `local_mode` would mean to just read the file locally which is what it's doing

That does make sense, though I'm still concerned with the terminology overlap there, especially if we want to support this through non-local mode for some reason. Maybe that wouldn't make sense though. I trust your judgment on the matter overall.

Reply:
I don't think it's possible to support this in non-local mode.